Added guide on server installation. Update to the readme.md file.

ktalap opened the pull request:
Updated the readme file with the information on how to start the server.

neiltalap committed:
Added guide on server installation. Update to the readme.md file.

Updated the readme file with the information on how to start the server.

oq-x commented:
You can install libdeflate from apt as well 
And homebrew

Skullians commented:
https://skullians.github.io/zeppelindocs/start/get-started.html This has existed for a while

neiltalap committed:
Connection limiter and bans

Needs testing, connection limiter and bans of the users. The bans are stored in a json file. Not tested yet.

ghost commented:
> https://skullians.github.io/zeppelindocs/start/get-started.html This has existed for a while

Just a small description for people to see, do some changes if you want to. I also implemented some bans and connection limiter. Contributions are welcome, what can I say.

ghost commented:
Why don't we just create a separate documentation from the organizations account? ZeppelinMC/docs. Use read the docs and it will be fire.

oq-x commented:
Thanks but I think a generic “playerlist manager” for whitelist, op, bans would work better

ktalap commented:
Some wasted effort right there. It's chill, I guess. Good luck with the project, will look into it 5 years down the road. Graveyard or not, may God be with you.

ghost commented:
Some wasted effort right there. It's chill, I guess. Good luck with the project, will look into it 5 years down the road. Graveyard or not, may God be with you.

Skullians commented:
@oq-x this should be merged. Seeing as my docs are not official you may as well have the installation guide somewhere people can actually easily access
